samples/azure-quantum/dynamic-error-correction/DynamicErrorCorrection.ipynb

```diff
@@ -0,0 +1,369 @@
+        "You can also check the status under Operations > Job management.\n",
+        "\n",
+        "**Understanding the results**<br/>\n",
+        "The histogram reports the ratio of the various outputs.</br>\n",
+        "For instance, if you get:</br>\n",
+        "- `Result: (1,0) Frequency: 0.92` means that in 92% of the total number of shots the operation returned a qubit with the value One and that no error correction was necessary.</br>\n",
+        "- `Result: (0,2) Frequency: 0.02` means that in 2% of the total number of shots the operation returned a qubit with the value Zero and that 2 error corrections were performed. The error correction failed as the value flipped.</br>\n",
+        "- `Result: (0,0) Frequency: 0` means that in 2% of the total number of shots the operation returned a qubit with the value One and that 2 error corrections were performed. The error correction failed as the value flipped.</br>\n",
+        "- `Result: (1,2) Frequency: 0.02` means that in 2% of the total number of shots the operation returned a qubit with the value One and that 2 error corrections were performed."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "%azure.target quantinuum.sim.h1-1e-preview\n",
+        "%azure.target-capability AdaptiveExecution\n",
+        "%azure.execute DynamicBitFlipCode shots=50 timeout=300 poll=10"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "%azure.status"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [],
+      "source": [
+        "%azure.output"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "# ↗ Run an advanced experiment - Iterative phase estimation\n",
+        "Now that you have learned the basics of Integrated hybrid, experiment with the Iterative phase estimation sample available in the sample gallery as well.</br>\n",
+        "To learn more, please refer to the [online documentation](https://aka.ms/AQ/Hybrid/Docs)."
+      ]
+    }
+  ],
+  "metadata": {
+    "kernel_info": {
+      "name": "python3"
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
```

Comment:
This seems to have the wrong kernel defined. Looking at code, this should be iqsharp kernel

Reply:
Fixed.